NetCord/Embed.cs

```diff
@@ -1,24 +1,81 @@
 ﻿namespace NetCord;
 
+/// <summary>
+/// Displays embedded content such as an image or URL, alongside a title and various other fields. All text in the embed must add up to under 6000 characters, and you can only have up to 10 embeds per message.
```

Comment:
The 6000 characters limit is for all embeds in a message, not for each embed.

Reply:
Didn't know that, will update alongside the provider documentation (discord's docs really don't cover most of this)